frontend/src/components/comments/CommentCard.jsx

```diff
@@ -107,6 +112,21 @@ export default function CommentCard({
           </div>
         )}
       </div>
+      {hasImage && (
+        <img
+          src={`data:${comment.imageType};base64, ${comment.image}`}
+          height="100%"
+          width="100%"
+        />
+      )}
+      {hasVideo && (
+        <ReactPlayer
+          url={require(`../../videos/${comment.videoName}`)}
```

Comment:
Please amend the file path to include the comment folder

`url={require(`../../videos/comment/${comment.videoName}`)}`

Reply:
fixed